man/coredns-sign.7

```diff
@@ -8,9 +8,9 @@
 .SH "DESCRIPTION"
 .PP
 The \fIsign\fP plugin is used to sign (see RFC 6781) zones. In this process DNSSEC resource records are
-added to the zone. The signatures that sign the resource records sets have an expiration date. This
-means the signing process must be repeated before this expiration data is reached. Otherwise the
-zone's data will go BAD (RFC 4035, Section 5.5). The \fIsign\fP plugin takes care of this.
+added. The signatures that sign the resource records sets have an expiration date, this means the
```

Comment:
This seems to be a grammar mistake to me.

Reply:
I didn't change this file. man/* files are added by github-ci. The only generated man file should be `man/coredns-bind ` file which I have changed the corresponding README. I don't know why ci changes these files for me. Maybe these are changes happened on master after I created PR and I need to rebase to HEAD of master branch.